datm/cime_config/config_component.xml

```diff
@@ -217,20 +218,22 @@
     <values match="last">
       <value compset="2000.*_DATM%1PT">$DATM_YR_START</value>
       <value compset="1850.*_DATM%QIA">1</value>
-      <value compset="1850.*_DATM%CRU">1</value>
+      <value compset="1850.*_DATM%CRUJRA2024">1</value>
+      <value compset="1850.*_DATM%CRUv7">1</value>
       <value compset="1850.*_DATM%GSW">1</value>
       <value compset="HIST.*_DATM%QIA">1895</value>
-      <value compset="HIST.*_DATM%CRU">1901</value>
+      <value compset="HIST.*_DATM%CRUJRA2024">1901</value>
+      <value compset="HIST.*_DATM%CRUv7">1901</value>
       <value compset="HIST.*_DATM%GSW">1901</value>
       <value compset="HIST.*_DATM%NLDAS2">$DATM_YR_START</value>
       <value compset="20TR.*_DATM%QIA">1895</value>
-      <value compset="20TR.*_DATM%CRU">1901</value>
+      <value compset="20TR.*_DATM%CRUJRA2024">1901</value>
+      <value compset="20TR.*_DATM%CRUv7">1901</value>
       <value compset="20TR.*_DATM%GSW">1901</value>
       <value compset="SSP.*_DATM%QIA">$DATM_YR_START</value>
-      <value compset="SSP.*_DATM%CRU">$DATM_YR_START</value>
+      <value compset="SSP.*_DATM%CRUJRA2024">$DATM_YR_START</value>
+      <value compset="SSP.*_DATM%CRUv7">$DATM_YR_START</value>
       <value compset="SSP.*_DATM%GSW">$DATM_YR_START</value>
-      <value compset="1850.*_DATM%CRU">1</value>
```

Comment:
Ahh, these two can be removed, since they are duplicated at the top. Good catch.

Reply:
To be precise, the first (CRU) was eliminated, while the second (GSW) was a duplicate.